runtime/src/main/java/org/corfudb/runtime/clients/LayoutHandler.java

```diff
@@ -52,46 +59,116 @@ public LayoutClient getClient(long epoch, UUID clusterID) {
             .generateHandlers(MethodHandles.lookup(), this)
             .generateErrorHandlers(MethodHandles.lookup(), this);
 
+    /**
+     * TODO: Remove this method when Management RPC get landed.
+     */
+    @Deprecated
+    @ClientHandler(type = CorfuMsgType.LAYOUT_NOBOOTSTRAP)
+    private static Object handleNoBootstrap(CorfuMsg msg,
+                                            ChannelHandlerContext ctx, IClientRouter r)
+            throws Exception {
+        throw new NoBootstrapException();
```

Comment:
Can you delete these instead of deprecating ?

They are not exposed publicly and we dont support backward compatibility.

Reply:
Yeah they will be deleted after the following PRs get merged. Currently they are still used by some other CorfuMsg RPCs even if all the Layout RPCs has been replaced by Protobuf.